### ISSUE TITLE
Getting "The server is not starting..." After Installing Duniter by the Steps on Wiki

### ISSUE THREAD

**Issue by Idyllei** [duniter/duniter#504]
I followed the instructions on the Duniter wiki for installing Duniter on Windows. However, every time I open it, I am told that the server is not starting.

I have no logs being produced.
I am using v0.20.1 of Duniter.
I ran `node .\duniter` from the `Duniter\sources\bin` directory and got the following:
```
C:\Program Files (x86)\Duniter\sources\bin>node .\duniter
module.js:327
    throw err;
    ^

Error: Cannot find module 'C:\Program Files (x86)\Duniter\sources\node_modules\sqlite3b\lib\binding\Release\node-v46-win32-x64\node_sqlite3.node'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Program Files (x86)\Duniter\sources\node_modules\sqlite3b\lib\sqlite3.js:4:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
```

Is there a way to fix this?

**Comment by M5oul** [comment]
Similar to this one https://github.com/duniter/duniter/issues/381#issue-145647211.

**Comment by c-geek** [comment]
Fixed in 0.20.2.
